### PR TITLE
[i18n] Spanish translation

### PR DESCRIPTION
**Pull request by pgaetani**
Initial spanish .po files.

### CODE REVIEW

**Comment by tekulvw** [comment]
What about `core/locales`?

**Comment by pgaetani** [comment]
Just noticed them, was actually doing those right now. Will be committed soon.

**Comment by pgaetani** [comment]
Done deal.

File: cogs/general/locales/es.po [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+msgstr "{} ¿Tal vez más que 1? ;P"
+
+#: ../general.py:91
+msgid ""
+"Nice try. You think this is funny?How about *this* instead:\n"
+"\n"
+msgstr ""
+"Buen intento. ¿Te parece gracioso? Qué tal *esto* mejor:\n"
+"\n"
+"\n"
```
Comment (tekulvw) [inline cogs/general/locales/es.po]: One too many newlines

**Review by tekulvw — CHANGES_REQUESTED** [review]

File: cogs/general/locales/es.po (hunk 1 above)
Comment (pgaetani) [reply cogs/general/locales/es.po]: Deleted now.

**Review by tekulvw — APPROVED** [review]

**Pull request merged**